--- a/adapter/parser.go
+++ b/adapter/parser.go
@@ -173,6 +173,13 @@
 			break
 		}
 		proxy, err = outbound.NewTrustTunnel(*trustTunnelOption)
+	case "tailscale":
+		tailscaleOption := &outbound.TailscaleOption{BasicOption: basicOption}
+		err = decoder.Decode(mapping, tailscaleOption)
+		if err != nil {
+			break
+		}
+		proxy, err = outbound.NewTailscale(*tailscaleOption)
 	default:
 		return nil, fmt.Errorf("unsupport proxy type: %s", proxyType)
 	}
